ui/scss/component/_button.scss

```diff
@@ -242,6 +242,7 @@
 }
 
 .button__label {
+  max-width: 10rem;
```

Comment:
I would like to avoid adding this to the generic button style. Can you add it here instead?
https://github.com/lbryio/lbry-desktop/blob/master/ui/scss/component/_comments.scss#L195

Reply:
Thank you @seanyesmunt , I've changed it, I hope it's ok.